crates/api_crud/src/comment/update.rs

```diff
@@ -77,6 +80,17 @@ impl PerformCrud for EditComment {
       .await?;
     }
 
+    if let Some(language_id) = data.language_id {
```

Comment:
Might as well make a helper function for `Option<LanguageId`, since you use this code both here and in UpdatePost.

Reply:
done